java/code/src/com/suse/manager/webui/websocket/Notification.java

```diff
@@ -59,8 +61,13 @@
      */
     @OnOpen
     public void onOpen(Session session, EndpointConfig config) {
-        User user = (User)session.getUserProperties().get("currentUser");
         if (session != null) {
+            User user = Optional.ofNullable(session.getUserProperties().get("webUserID"))
+                    .map(webUserID -> {
+                        User innerUser = UserFactory.lookupById((Long) webUserID);
+                        innerUser.getRoles();
+                        return innerUser;
+                    }).orElse(null);
```

Comment:
I think i commented about this before in a previous version but can we use `ifPresent` here instead of assigning null to user?

Reply:
Thanks @lucidd. I just change it.